pkg/3scale/amp/operator/redis_reconciler.go

```diff
@@ -72,6 +76,25 @@ func (r *RedisReconciler) Reconcile() (reconcile.Result, error) {
 		return reconcile.Result{}, err
 	}
 
+	// 3scale 2.14 -> 2.15 Upgrade
+	// delete NAMESPACE attribute from secret system-redis
+	// and delete REDIS_NAMESPACE env var from deployments system-app and system-sidekiq
+	res, err := r.deleteSystemRedisSecretNamespaceAttribute()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	if res.Requeue {
```

Comment:
Requeue is always nil, no need for handling it since we will never get here.

Reply:
Done. Thank you